web/text_layer_builder.js

```diff
@@ -321,11 +323,18 @@ var TextLayerBuilder = (function TextLayerBuilderClosure() {
     _bindMouse: function TextLayerBuilder_bindMouse() {
       var div = this.textLayerDiv;
       var self = this;
+      var expandDivsTimer = null;
       div.addEventListener('mousedown', function (e) {
         if (self.enhanceTextSelection && self.textLayerRenderTask) {
           self.textLayerRenderTask.expandTextDivs(true);
-          return;
+//#if !(MOZCENTRAL || FIREFOX)
+        if (expandDivsTimer) {
```

Comment:
The indenting is off here. This needs two more spaces, as well as the lines below, including the `return` and `}`. The preprocessor tags are fine like this and should not be indented.